### PR TITLE
Commit database cleared content

### PR DESCRIPTION
**Pull request by isra17**

### CODE REVIEW

File: frontera/contrib/backends/sqlalchemy/__init__.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def __init__(self, manager):
             session = self.session_cls()
             for name, table in DeclarativeBase.metadata.tables.items():
                 session.execute(table.delete())
+            session.commit()
```
Comment (sibiryakov) [inline frontera/contrib/backends/sqlalchemy/__init__.py]: Are you sure that DDL statement needs commit?

Comment (isra17) [inline frontera/contrib/backends/sqlalchemy/__init__.py]: At least on Postgres it does. Without this line, `SQLALCHEMYBACKEND_CLEAR_CONTENT` doesn't work.

Comment (sibiryakov) [inline frontera/contrib/backends/sqlalchemy/__init__.py]: ok

**Comment by sibiryakov** [comment]
Thank you!

**Pull request merged**